.github/workflows/ci.yml

```diff
@@ -101,3 +101,30 @@ jobs:
       - name: Teardown integration environment
         run: npm run testenv:stop
         if: always()
+
+  build-publish:
+    runs-on: ubuntu-latest
+    needs:
+      - lint
+      - test
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
+          persist-credentials: false
+
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v4
+        id: semantic
+        # Only run on non-PR events or only PRs that aren't from forks
+        if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
```

Comment:
This can just be set to `${{ secrets.GITHUB_TOKEN }}` now, because Github now allows us to provide read and write permissions to the default GH token [here](https://github.com/hirosystems/api-toolkit/settings/actions).